lib/types/uuid.js

```diff
@@ -15,9 +20,11 @@ class Uuid {
      */
     constructor(buffer) {
         if (!buffer || buffer.length !== 16) {
-            throw new Error("You must provide a buffer containing 16 bytes");
+            throw new TypeError(
+                "You must provide a buffer containing 16 bytes",
+            );
```

Comment:
Does this suit `TypeError`? In Python, I'd rather use `ValueError`.

Reply:
There is no ValueError in JS unforunately (https://nodejs.org/api/errors.html). I think type error is the most suitable (or it can be just a generic error).

Reply:
OK, then let's stay with TypeError.